extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -476,10 +476,12 @@ export class AmpA4A extends AMP.BaseElement {
         .then(fetchResponse => {
           checkStillCurrent(promiseId);
           this.protectedEmitLifecycleEvent_('adRequestEnd');
-          // If the response has response code 204, or is null, collapse it.
-          if (!fetchResponse
-              || !fetchResponse.arrayBuffer
-              || fetchResponse.status == 204) {
+          if (!fetchResponse) {
```

Comment:
The logic in here is getting thorny enough to worry me.  :-P  Let's add a comment here explaining this case and the intended behavior from returning null.